client/elements/suttaplex/sc-parallel-item.html

```diff
@@ -19,7 +19,7 @@
 
       .parallel-item-main-info-container {
         width: 100%;
-        max-width: 320px;
+        max-width: 360px;
```

Comment:
This should be tested on mobile devices in chrome devtools - particularly if it fits inside the smallest devices' viewports and doesn't blow up the layout horizontally.

Reply:
I have tested it and it seems fine.